lib/Controller/ChatController.php

```diff
@@ -468,6 +468,11 @@ public function receiveMessages(int $lookIntoFuture,
 		$i = 0;
 		$messages = $commentIdToIndex = $parentIds = [];
 		foreach ($comments as $comment) {
+			$message = $comment->getMessage();
+			if ($this->messageParser->isSharedFile($message) && !$this->messageParser->fileOfMessageExists($message)) {
+				continue;
+			}
```

Comment:
```suggestion
```

This endpoint should work as before.

Only those 2:
* getObjectsSharedInRoomOverview
* getObjectsSharedInRoom

Should filter them out

Reply:
`receiveMessages` which is the normal chat view. And we have intergration tests for it and they all fail now.